src/_zkapauthorizer/tests/foolscap.py

```diff
@@ -33,13 +32,13 @@ def echo(argument=Any()):
         return Any()
 
 
-@implementer(RIStorageServer)
-class StubStorageServer(object):
-    pass
+class StubStorageBackend(object):
```

Comment:
I'd be inclined to keep the name, given that this is a stub `StorageServer` implementation.